retrofit/src/test/java/retrofit2/RetrofitTest.java

```diff
@@ -171,6 +171,37 @@
     assertSame(callFactory, two.callFactory());
   }
 
+  @Test public void builtInConvertersAbsentInCloneBuilder() {
+    Retrofit retrofit = new Retrofit.Builder()
+        .baseUrl(server.url("/"))
+        .build();
+
+    assertEquals(retrofit.converterFactories().size() - 1,
+        retrofit.newBuilder().converterFactories().size());
+  }
+
+  @Test public void builtInConvertersAddedOnBuild() {
+    Retrofit.Builder builder = new Retrofit.Builder();
+    Retrofit retrofit = builder
+        .baseUrl(server.url("/"))
+        .build();
+
+    assertEquals(builder.converterFactories().size() + 1, retrofit.converterFactories().size());
+  }
+
+  @Test public void builtInConvertersRemainFirstInClone() {
```

Comment:
I think the test you really want is that the converterFactories on the builder is mutable and initially empty

Reply:
b2b3b841 has got the second part of that covered. 

Mutability seems like something that's an implementation detail. Thoughts?